tests/e2e/rbac_privileges_test.go

```diff
@@ -178,7 +178,7 @@ var _ = Describe("RBAC Privileges", func() {
 				{
 					APIGroups: []string{""},
 					Resources: []string{"secrets"},
-					Verbs:     []string{"list", "watch", "create", "delete"},
+					Verbs:     []string{"list", "watch", "create", "delete", "update"},
```

Comment:
please also update this permission in `management-plane-charts` lifecycle-manager/files/roles/klm-watcher-certmanager-role.yaml before release